javagen/src/main/java/com/azure/autorest/model/clientmodel/GenericType.java

```diff
@@ -159,6 +159,11 @@ public final String defaultValueExpression(String sourceExpression) {
         return sourceExpression;
     }
 
+    @Override
+    public String defaultValueExpression() {
+        return "null";
```

Comment:
We might actually want to return empty collection when it is null from JSON. But avoid this unless necessary as this could be big behavior change.